data/vocab/scholar.ttl

```diff
@@ -469,6 +469,36 @@ scholar:mostCitedWork a rdf:Property ;
     rdfs:range lite:Work ;
     rdfs:domain lite:Person .
 
+scholar:totalCoAuthoredWorks a rdf:Property ;
+    rdfs:label "total co-authored works"@en ;
+    rdfs:comment "The total number of works a person has co-authored."@en ;
+    rdfs:domain lite:Person ;
+    rdfs:range xsd:integer .
+
```

Comment:
This is a property of the edge. Not applicable to the lite:Person as a property. We can handle this use case via named graph or rdf reification.

```suggestion
```